### PR TITLE
add base_volatility_spread to feed_configuration

### PR DESCRIPTION
**Pull request by NNagato**

### CODE REVIEW

File: data/storage/bolt.go [hunk 1]
```diff
@@ -127,9 +129,41 @@ func NewBoltStorage(path string) (*BoltStorage, error) {
...
+			if dataBytes := b.Get([]byte(feed)); dataBytes == nil {
+				var fi feedInfo
+				if dataBytes := bdf.Get([]byte(feed)); dataBytes == nil {
+					fi.Enabled = true
+				}
+				dataJSON, err := json.Marshal(fi)
+				if err != nil {
+					return err
+				}
+				return b.Put([]byte(feed), dataJSON)
```
Comment (secmask) [inline data/storage/bolt.go]: return here will abort migrate on first success feed?

File: data/storage/bolt.go [hunk 2]
```diff
@@ -127,9 +129,41 @@ func NewBoltStorage(path string) (*BoltStorage, error) {
 		db: db,
 		l:  zap.S(),
 	}
+	if err = storage.migrateFeedConfiguration(); err != nil {
```
Comment (secmask) [inline data/storage/bolt.go]: please add a todo to remove this migrate after it get deploy and run normally, as it no longer need.

File: data/storage/bolt.go (hunk 1 above)
Comment (NNagato) [reply data/storage/bolt.go]: fixed in https://github.com/KyberNetwork/reserve-data/pull/784/commits/9e80c4ef75fe9cf7189b51b24142e8bd505935dc

File: data/storage/bolt.go (hunk 2 above)
Comment (NNagato) [reply data/storage/bolt.go]: updated in https://github.com/KyberNetwork/reserve-data/pull/784/commits/284b376f7a312e2c982235168fd460163575a936

**Pull request merged**